### PR TITLE
DOCS-10185 pre-install for 1.77

### PR DESCRIPTION
**Pull request by ferferrarazzo**

### CODE REVIEW

File: modules/ROOT/pages/update-acm.adoc [hunk 1]
```diff
@@ -88,6 +89,17 @@ If you are upgrading from a package version before 1.74, follow these steps:
...
+[[pre-177]]
+== Pre-Install Steps When Upgrading from a Version before 1.77
+
+If you are upgrading from a package version before 1.77, follow these steps:
+
+. Open the *Developer Console* and open the *Query Editor*.
+. Enter `SELECT Id, Name, Owner.Name, Owner.IsActive FROM acm_pkg__CommunityApi__c WHERE Owner.IsActive = false` and click *Execute*.
+. If the query lists any users, change their status to Active with the https://help.salesforce.com/articleView?id=000336144&type=1&mode=1[reactivate a user] instructions.
+
+Return to the main list of upgrade instructions and continue to follow them.
```
Comment (hannanelson) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
Return to the main list of upgrade instructions to continue.
```

File: modules/ROOT/pages/update-acm.adoc [hunk 2]
```diff
@@ -88,6 +89,17 @@ If you are upgrading from a package version before 1.74, follow these steps:
...
 Return to the main list of upgrade instructions and continue to follow them.
 
+[[pre-177]]
+== Pre-Install Steps When Upgrading from a Version before 1.77
+
+If you are upgrading from a package version before 1.77, follow these steps:
+
+. Open the *Developer Console* and open the *Query Editor*.
+. Enter `SELECT Id, Name, Owner.Name, Owner.IsActive FROM acm_pkg__CommunityApi__c WHERE Owner.IsActive = false` and click *Execute*.
+. If the query lists any users, change their status to Active with the https://help.salesforce.com/articleView?id=000336144&type=1&mode=1[reactivate a user] instructions.
```
Comment (hannanelson) [inline modules/ROOT/pages/update-acm.adoc]: Font for Active OK?

File: modules/ROOT/pages/update-acm.adoc [hunk 3]
```diff
@@ -26,6 +26,7 @@ Attempting to upgrade the package without this feature enabled causes the proces
 . Verify the version number and name of the release.
 . If you are upgrading from a package version before 1.26, follow the <<pre-126>> and then continue with these instructions.
 . If you are upgrading from a package version before 1.74, follow the <<pre-174>> and then continue with these instructions.
+. If you are upgrading from a package version before 1.77, follow the <<pre-177>> and then continue with these instructions.
```
Comment (hannanelson) [inline modules/ROOT/pages/update-acm.adoc:29]: I always use earlier/later than, but I don't see anything forbidding this in the SG.

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/update-acm.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Good point. UI says "Active" but I think I'll just use "active" and make it part of the sentence.

**Review by hannanelson — APPROVED** [review]

**Pull request merged**